recipes/srwpy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "srwpy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Synchrotron-Radiation-Workshop/SRW/archive/v{{ version }}.tar.gz
+  sha256: 53bdcb0c02c59659bd0cb3849eac15c166db65b69d6fb62d4b8bf97cc4c96be8
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - fftw  # [win]
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - fftw  # [win]
+    - matplotlib-base
+    - numpy
+    - pillow
+    - scipy
+
+test:
+  imports:
+    - srwpy
+    - srwpy.srwl_bl
+    - srwpy.srwlib
+    - srwpy.srwlpy
+    - srwpy.uti_math
+
+about:
+  home: https://github.com/Synchrotron-Radiation-Workshop/SRW
+  license: LicenseRef-SRW
```

Comment:
```suggestion
  license: EPICS
```

Reply:
This code has nothing to do with EPICS.

Reply:
It's not, but the license is named EPICS by SPDX. See https://spdx.org/licenses/EPICS.html

Reply:
Changed via https://github.com/conda-forge/staged-recipes/pull/14021/commits/d519740417b16d4e9c0b19fc09fa725a21bf50b7